app/views/hyrax/base/_show_actions.html.erb

```diff
@@ -35,4 +41,6 @@
 
 <!-- COinS hook for Zotero -->
   <span class="Z3988" title="<%= export_as_openurl_ctx_kev(presenter) %>"></span>
+  <%= render 'hyrax/dashboard/collections/form_for_select_collection', user_collections: @user_collections %>
```

Comment:
Can you add a blank line and a comment before this render with a comment stating `  <!-- Render Modals -->`?

Reply:
I notice that the modal renders with all the content right justified.  The same modal renders left justified when accessed from Dashboard -> Works -> click checkboxes -> Add to Collection.  Can you explore why they render differently?

Reply:
got it!